### ISSUE TITLE
Latest 2.0-rc1 branch gives error on jrnl execution

### ISSUE THREAD

**Issue by notbalanced** [jrnl-org/jrnl#456]
Pulled down the latest from the 2.0-rc1 branch and got the following error when trying to run jrnl:

```
Traceback (most recent call last):
  File "C:\Util\Python27\lib\runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "C:\Util\Python27\lib\runpy.py", line 72, in _run_code
    exec code in run_globals
  File "d:\Projects\jrnl-nb\jrnl\__main__.py", line 4, in <module>
    from . import cli
  File "jrnl\cli.py", line 15, in <module>
    from . import plugins
  File "jrnl\plugins\__init__.py", line 13, in <module>
    from .template_exporter import __all__ as template_exporters
  File "jrnl\plugins\template_exporter.py", line 49, in <module>
    __all__.append(__exporter_from_file(template_file))
  File "jrnl\plugins\template_exporter.py", line 42, in __exporter_from_file
    "template": template
TypeError: type() argument 1 must be string, not unicode
```

**Comment by radomirbosak** [comment]
@notbalanced as workaround, try to install `jrnl` using `pip3` instead of `pip`.

I could reproduce your error when using `pip` for installation. What did you use for your installation of the `2.0-rc1` branch?

**Comment by wren** [comment]
Should be fixed on v2.0-rc2+. Please reopen if you're still seeing this.
